src/components/markdown-editor/index.js

```diff
@@ -20,7 +20,7 @@ const converter = new Showdown.Converter({
   tasklists: true
 });
 
-const MarkdownEditor = () => {
+const MarkdownEditor = (props) => {
```

Comment:
instead of 
```javascript
const MarkdownEditor = (props) => {
```
we should use 
```javascript
const MarkdownEditor = ({edition}) => {
```

Reply:
i just readed in the other view, we can use disabled as a boolean property insted of `edition`